### PR TITLE
Added API docs about the flush effect

### PR DESCRIPTION
**Pull request by Andarist**

### CODE REVIEW

File: docs/api/README.md [hunk 1]
```diff
@@ -539,6 +539,32 @@ function* takeOneAtMost() {
 }
 ```
 
+### `flush(channel)`
+
+Creates an effect that instructs the middleware to flush all buffered items from the channel. Flushed items are returned back to the saga, so can they be utilized.
```
Comment (smashercosmo) [inline docs/api/README.md]: Probably, "so they can be"

Comment (Andarist) [inline docs/api/README.md]: What do you mean? Just indicating that they can be utilized by the user if he wants to and thats it not mandatory?

Comment (smashercosmo) [inline docs/api/README.md]: I mean the word order. I could be wrong, cause I'm not a native speaker) but I thought that verb should go after subject.

Comment (Andarist) [inline docs/api/README.md]: ah, ofc, you are right, didnt catch it while second reading, thanks!

**Comment by yelouafi** [comment]
thanks

**Pull request merged**